proto/metalstack/api/v2/image.proto

```diff
@@ -148,17 +148,8 @@ message ImageQuery {
     min_len: 2
     max_len: 128
   }];
-  // Url of the image to query
```

Comment:
This was not deleted in main ?

Reply:
No, I guess it wasn't. Don't know how that happened :smile: